assets/dev/scss/frontend/_global.scss

```
@@ -291,19 +291,11 @@
 	}
 }
 
-.elementor-widget-e-heading .e-heading-base a,
-.elementor-widget-e-paragraph .e-paragraph-base a {
-	color: unset;
-	font-family: unset;
-	font-size: unset;
-	font-weight: unset;
-	text-transform: unset;
-	font-style: unset;
-	text-decoration: unset;
-	line-height: unset;
-	text-align: unset;
-	letter-spacing: unset;
-	text-indent: unset;
+.elementor {
+	.e-heading-base a,
+	.e-paragraph-base a {
```

maybe for the sake of reducing the outreach for this quite general selector, could we add a more specific targeting of anchor tags like so -

```suggestion
	.e-heading-base > a,
	.e-paragraph-base > a {
```